get-started/6-learn-more.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
FYI, much of this work is duplicated in the new flutter.io/docs page (https://github.com/flutter/website/pull/817). 

If you want, we could just link to that, and set up Google Analytics to track the funnel strictly (only people coming from codelab to the docs page count to the funnel's last step).

Reply:
I have not heard of that page before now (and don't see it staged anywhere), so hard for me to comment on that. The goal of this last `learn more` step is not to offer an list of all docs; it's to offer a short, opinionated list of likely next steps.